### ISSUE TITLE
Extracting tar.gz/tar.bz2 in one-step

### ISSUE THREAD

**Issue by wapmorgan** [Gemorroj/Archive7z#23]
https://superuser.com/questions/80019/how-can-i-unzip-a-tar-gz-in-one-step-using-7-zip

**Comment by Gemorroj** [comment]
I think we can do it by configurable parameter. Like `OVERWRITE_MODE_*`.

**Comment by wapmorgan** [comment]
No, it requires two commands to get list of files/to extract - like `7z x -so ../UnifiedArchive.tar.gz | 7z l -si -ttar`. And this is required for all operations
